Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md

```diff
@@ -0,0 +1,78 @@
+{{{
+  "title": "Anomaly Detection & Forecasting",
+  "date": "02-13-2018",
+  "author": "Jason Oldham",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": true
+}}}
+
+
+### Overview:
+
+This document will cover anomaly detection, confidence bands, and forecasting functionality accessed via the Graphs tab in the Cloud Application Manager Monitoring site.  Our anomaly detection and forecasting is based on the Holt Winters forecasting model.
+
+Holt Winters (Triple Exponential Smoothing)
+
+•	Well known forecasting model
+•	Considers three components of dataset
+     o	 Level: Behavior of the y-axis value over time
+     o	Trend: Behavior of the incline/slope of the dataset over time
+     o	Season: Repeating patterns in a dataset over a fixed time period
+
+ ![Holt Winters](../../images/HoltWinters.PNG)
+
+
+#### Features:
+
+•	Can be applied to CloudWatch datasets and Watcher telemetry data
+•	You do not need to be a managed account to access graphs
+•	Customer must grant Cloud Application Manager access to see their CloudWatch data
+     o	Provider’s ARN contains ReadOnlyAccess permission
+•	The forecasting function can be projected arbitrarily in the future
+•	Ability to copy the metric ID to clipboard
+•	Intuitive legend to allow users to apply functions to the corresponding datasets on the graph
+
+**Forecast** – Graphs based on projected future states of metric data.  Future time periods can be specified for any metric (including those originating from AWS CloudWatch).
+
+**Confidence** – A tool to calibrate the uncertainty in a forecasted dataset.  This allows for the visualization of anomalies as a signal crosses over a confidence interval.
+
+**Anomaly Detection** – Visual representation of the degree of aberration within a signal.
+
+
+#### Navigation 
+
+**Forecast:**
+To access the anomaly detection and forecasting function you first need to go to the Graphs tab.
+
+
+ ![Graphs Left Nav](../../images/GraphsLeftNav.PNG)
+
+From there you will select your Source and then the metrics you want to view.
+
+![Graph Source](../../images/Graphsource.PNG)
```

Comment:
change to `(../../images/cloud-application-manager/GraphSource.PNG)`
[notice the capital "S"]